### ISSUE TITLE
"Invalid typeof comparison value" warning when using backtick strings

### ISSUE THREAD

**Issue by 1e1001** [denoland/deno_lint#741]
### Lint Name

valid-typeof
### Code Snippet

```ts
typeof `test` === `string`;
```

### Expected Result
no warning
### Actual Result
warning on the \`string`, "Invalid typeof comparison value deno-lint(valid-typeof)"
### Version
deno 1.11.1 (release, x86_64-unknown-linux-gnu)
v8 9.1.269.35
typescript 4.3.2
